server/src/infra/configuration.rs

```diff
@@ -89,6 +90,8 @@ pub struct HttpUrl(pub Url);
 #[derive(Clone, Deserialize, Serialize, derive_builder::Builder, derive_more::Debug)]
 #[builder(pattern = "owned", build_fn(name = "private_build"))]
 pub struct Configuration {
+    #[builder(default = r#"PathBuf::from("./app")"#)]
+    pub assets_path: PathBuf,
```

Comment:
Now that's a bit too high :D
Let's put it right after the key seed. It affects the order in which the debug info is printed, and I want to highlight the most important/common config values at the top. This is more niche.